tests/pr-milestone.test.ts

```diff
@@ -145,4 +145,16 @@ describe("PR milestone checks", () => {
             expect(dm.warn).not.toHaveBeenCalled();
         })
     }
+
+    it("does not warn the milestone is closing in 4 days or less and PR is merged", async () => {
+        var closeDate = new Date();
+        closeDate.setDate(closeDate.getDate() + 4);
```

Comment:
Just a suggestion to avoid using the value exactly at the limit value `4` that we use between "closing soon" and "not closing soon" (in risk of hitting edge case of `<` vs `≤` unexpectedly and making the test actually test "not closing soon" by mistake – even if in that case that shouldn't change anything).

```suggestion
        closeDate.setDate(closeDate.getDate() + 2);
```

Just reduces the risk of doubt when reading the test code imho; but not an issue per se, so feel free to ignore 🤷